app/views/govuk_publishing_components/components/docs/input.yml

```diff
@@ -56,3 +56,10 @@ examples:
         text: "Search for"
       name: "name"
       value: "moose"
+  autofocused:
+    data:
```

Comment:
totally non-blocking: you could add a 'description' to this to make a note about user research like you have done in this pull request summary

Reply:
I'm using this to replicate the current setting of the Signon login page behaviour, which I've been told have been tested with users. I'd like to add more details, but I don't have 😞 .

Reply:
I meant just that the description says be careful with autofocus